query/src/main/resources/application.conf

```diff
@@ -31,3 +31,12 @@ db {
   }
 }
 
+cassandra {
+  host = "127.0.0.1"
+  host = ${CASSANDRA_CLUSTER_HOSTS}
+  port = 9142
```

Comment:
This is a mistake from a previous try. `CassandraUnit` uses 9142, but we're not using it when running the application. Will change back to default port